src/main/java/com/rakovets/course/java/core/practice/operators/Task02.java

```diff
@@ -32,6 +32,6 @@ public static void main(String[] args) {
     static int getAmountRemainingCargo(int cargo, int carryingCapacity) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        return cargo%carryingCapacity;
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"